src/app/layout.tsx

```diff
@@ -16,7 +24,24 @@ export default function RootLayout({
 }) {
   return (
     <html lang="en">
-      <body className={inter.className}>{children}</body>
+      <body className={jakartaSans.className}>
+        <ThemeRegistry>
+          <CssBaseline />
+          <Sidebar />
+
+          <Box
+            sx={{
+              gap: '0px',
```

Comment:
@asad-codingcrafts 
Try to use Theme.spacing(3) instead of hardcoding values in pixels.
For padding and margin.

#INFO
Theme.spacing(1) means 8px.
This will help in standardizing UI and consistent padding and margins across the app